questionnaires/superset/README.md

```diff
@@ -112,3 +112,46 @@ docker-compose up
 ```
 docker-compose up
 ```
+
+## Share Dashboard
+
+### How to Share Dashboard Publicly
+
+Anyone with the link will be able to view the dashboard.
+
+- Click "Settings" menu on upper right corner
+- Click on "List Roles" menu item
+- Click on "+" icon to add a new role
+- Add following permissions and save
+  - can read on CssTemplate
+  - can read on Chart
+  - can read on Dashboard
+  - can read on DashboardPermalinkRestApi
+  - can dashboard permalink on Superset
+  - can dashboard on Superset
+  - can explore json on Superset
+  - datasource access on [dataset-used-for-dashboard]
+
```

Comment:
Be very careful when setting the "datasource access" permission. Make sure to choose a dataset you know can be public. If you name your public datasets starting with "PUBLIC" it will be easy to not accidentally make a private dataset public.